doc/user/content/concepts/sinks.md

```diff
@@ -21,6 +21,16 @@ change data capture (CDC) producers for the given source or view.
 Currently, Materialize only supports sending sink data to Kafka. See
 the [Kafka sink documentation](/sql/create-sink/kafka) for details.
 
+## Clusters and sinks
```

Comment:
https://preview.materialize.com/materialize/32226/concepts/sinks/#clusters-and-sinks